core/src/main/java/io/confluent/kafka/schemaregistry/rest/resources/CompatibilityResource.java

```diff
@@ -137,11 +141,22 @@ public void testCompatibilityBySubjectName(
       throw Errors.schemaRegistryException(
```

Comment:
A few lines above this line, we catch `InvalidSchemaException`.  Instead of throwing an exception, let's put the error message from `InvalidSchemaException` in a `CompatibilityCheckResponse` and return that.